reframe/core/environments.py

```diff
@@ -177,6 +177,15 @@ def __repr__(self):
                           self.modules, self.variables)
 
 
+def format_flags(flags):
```

Comment:
I would move that inside the `details()` function. It's not meant to be used anywhere else. Other than that, looks good to me.

Reply:
You should now remove this function.